apps/styleguide/src/display/LinksList.vue

```diff
@@ -32,13 +32,6 @@ export default {
 <docs>
 Display a list of links
 ```
-const links = [
-    { label: 'Molgenis', href: 'https://molgenis.org', },
-    { label: 'Vue',  href: 'https://vuejs.org/' } 
-]
-<template>
-  <links-list :items="links" ><p></p>Sample content</p></<links-list>
-</template>
-
+<links-list :items="[{label: 'Molgenis', href: 'https://molgenis.org'}]"/>
```

Comment:
single item list is not a great example of a list rendering component

Reply:
indeed, will add another item before merging :-)